web/controllers/page_controller.ex

```diff
@@ -1,5 +1,6 @@
 defmodule Booky.PageController do
   use Booky.Web, :controller
+  plug Booky.Plug.AuthenticateUser
```

Comment:
Can this go in the router instead? Or whoever adds controllers in future might forget to put this in.